packages/cacti-plugin-copm-corda/sample/src/main/kotlin/com/copmCorda/SampleInteropConfig.kt

```diff
@@ -0,0 +1,119 @@
+package com.copmCorda
```

Comment:
Why is this file is a sample?

Reply:
This is in the `sample` subfolder within the Corda plugin folder.

Reply:
This needs to be reviewed whether a `SampleInteropConfig` is required as the code seems general. Please open an issue to track this?

Reply:
The application-specific code is in the companion object.  As with [above](https://github.com/hyperledger-cacti/cacti/pull/3625#discussion_r1860754213), this is a test implementation for running on the test docker image which has this data in the envrionment.  

It is deliberately set up as an abstract interface so that a production customer can do what they want -- with the info coming from json files or a future configuration system.